nrf_security/Kconfig.legacy

```diff
@@ -232,7 +269,7 @@ if MBEDTLS_LEGACY_CRYPTO_C
 
 config CC3XX_BACKEND
 	bool
-	# default y if CRYPTOCELL_USABLE
+	# default y if CaRYPTOCELL_USABLE
```

Comment:
Not only does it has commented out code, but also `CaRYPTOCELL_USABLE` got messed up in the process.